packages/contracts/contracts/foundry/SphinxPluginTypes.sol

```diff
@@ -75,6 +75,11 @@ struct ParsedAccountAccess {
     Vm.AccountAccess[] nested;
 }
 
+struct DeployedContractSize {
+    address account;
+    uint size;
+}
```

Comment:
I think this PR could've been simplified a decent amount by just using the `deployedCode` field of the `Vm.AccountAccess` type, which already exists in the `ParsedAccountAccess`

Reply:
Unfortunately, I found that `deployedCode` isn't reliably available I believe due to a bug in Foundry. If you want, I can send you the replication steps. To my knowledge, this doesn't cause a problem for us, but I was going to bring it up in the next sync. [I created a ticket for it.](https://linear.app/chugsplash/issue/CHU-657/investigate-if-missing-deployedcode-in-accountaccesses-could-cause)

I'll add a comment documenting this.

Reply:
Got it. We could probably just manually set the `deployedCode` field in the account accesses via `address(access.account).code`, assuming that this returns the correct bytecode

Reply:
Not important enough to block this PR, but just pointing that out